packages/dialect-react-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dialectlabs/react-ui",
-  "version": "1.0.0-beta.49",
+  "version": "1.0.0-beta.50",
```

Comment:
I would recommend bumping the version only after a release (so once merged to `master` and actually released). And the changes in the Changelog to be under the `UNRELEASED` section before actual release in order to avoid confusion.